react/features/stream-effects/blur/JitsiStreamBlurEffect.js

```diff
@@ -61,61 +65,108 @@ export default class JitsiStreamBlurEffect {
     }
 
     /**
-     * Loop function to render the background mask.
+     * Represents the run post processing.
      *
-     * @private
      * @returns {void}
      */
-    async _renderMask() {
-        if (!this._maskInProgress) {
-            this._maskInProgress = true;
-            this._bpModel.segmentPerson(this._inputVideoElement, {
-                internalResolution: 'low', // resized to 0.5 times of the original resolution before inference
-                maxDetections: 1, // max. number of person poses to detect per image
-                segmentationThreshold: 0.7, // represents probability that a pixel belongs to a person
-                flipHorizontal: false,
-                scoreThreshold: 0.2
-            }).then(data => {
-                this._segmentationData = data;
-                this._maskInProgress = false;
-            });
-        }
-        const inputCanvasCtx = this._inputVideoCanvasElement.getContext('2d');
+    runPostProcessing() {
+        this._outputCanvasCtx.globalCompositeOperation = 'copy';
 
-        inputCanvasCtx.drawImage(this._inputVideoElement, 0, 0);
-
-        const currentFrame = inputCanvasCtx.getImageData(
+        // draw segmentation mask
```

Comment:
Punctuation: start with a capital, end with a period, please. (here and on every other code comment)